Nukkit/src/main/resources/plugin.yml

```diff
@@ -119,6 +119,7 @@ permissions:
       plots.toggle: true
       plots.toggle.titles: true
       plots.toggle.chat: true
+      plots.toggle.time: true
```

Comment:
Is redundant, breaking doesn't build Nukkit artifacts :P

Reply:
In the case that this ever happens, this doesn't hurt. Not that it will, but let's pretend it might.